test/cli.js

```diff
@@ -0,0 +1,31 @@
+import execa from "execa";
+import path from "path";
+
+import initFixture from "./helpers/initFixture";
+
+const BIN = path.join(__dirname, "../bin/lerna.js");
+const bin = (args, options) => execa(BIN, args, options);
+
+describe("cli", () => {
+  it("should throw without command", async () => {
+    try {
+      await bin([]);
+      fail(`Expected an exception, but it didn"t throw anything`); // eslint-disable-line
```

Comment:
Where does `fail` come from? Seems a little odd to templatize a string that isn't interpolated.

Reply:
`fail` is exposed by jasmine. I changed those to match the jest recommendations: https://facebook.github.io/jest/docs/en/tutorial-async.html#async-await

Reply:
Ah, that's why I didn't recognize it. I really wish we could upgrade to Jest 21, but something in Jest 20 broke how we run the integration tests...